webservices/wsgi/wsgiwps.py

```diff
@@ -54,7 +54,7 @@ def application(environ, start_response):
 
     # create the WPS object
     try:
-        wps = pywps.Pywps(environ["REQUEST_METHOD"])
+        wps = pywps.Pywps(environ)
```

Comment:
pass the entire request environment instead of the request method only, so that we can grab settings from there.
this is the way to pass in variables from nginx as nginx is not starting the wsgi app by itself.